### PR TITLE
Fix #72: PUT Request Fails

### LINKED ISSUE #72: PUT request fails for events
`updateObjectFromXML() got an unexpected keyword argument 'xmlObject'`

Call to `updateObjectFromXML` probably needs to be modified to work with the updated function in codalib. Tests for the PUT are failing but are marked with `xfail`.

### PR DESCRIPTION
**Pull request by runderwood**
@ldko @somexpert  Pretty sure I did this on a deleted branch somewhere. In any case, PUTs seem to work and the tests pass.

Closes #72

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -44,9 +46,10 @@ def entry_xml(self):
               </content>
             </entry>
         """
-        self.attributes["event_date_time"] = xsDateTime_format(
-            self.attributes["event_date_time"]
-        )
+        if isinstance(self.attributes['event_date_time'], datetime.datetime):
+            self.attributes["event_date_time"] = xsDateTime_format(
+                self.attributes["event_date_time"]
+            )
```
Comment (ldko) [inline tests/conftest.py]: This `if` and this file has inconsistent single quote/double quote usage around `attributes` keys

File: premis_event_service/presentation.py [hunk 2]
```diff
@@ -33,6 +33,21 @@
     "linkingAgentIdentifier", "linkingAgentIdentifierValue"
 ]
 
+xpath_map = collections.OrderedDict()
+xpath_map['@namespaces'] = PREMIS_NSMAP
+xpath_map["event_identifier_type"] = 'premis:eventIdentifier/eventIdentifierType'
+xpath_map["event_identifier"] = 'premis:eventIdentifier/premis:eventIdentifierValue'
```
Comment (ldko) [inline premis_event_service/presentation.py]: single quotes and double quotes around `xpath_map` keys.

File: tests/conftest.py (hunk 1 above)
Comment (runderwood) [inline tests/conftest.py]: This repo is full of double quotes.

Comment (ldko) [inline tests/conftest.py]: I know this repo mixes quotes. I don't expect you to go through the files and change them. On new changes, it makes sense to me to be consistent at least within the changeset. So, if line 49 used double quotes like in 50 and 51, I wouldn't have mentioned it.

Comment (runderwood) [inline tests/conftest.py]: Sorry! Didn't mean I don't want to be consistent. Just meant I forget to check when I wrap stuff!

**Pull request merged**